editoast/src/main.rs

```diff
@@ -61,6 +62,17 @@ async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
         Commands::ImportRailjson(args) => import_railjson(args, pg_config),
     }
 }
+
+/// Config to share in a rocket state to build some responses
+#[derive(Derivative)]
+#[derivative(Default)]
+pub struct LayersConfig {
+    #[derivative(Default(value = "18"))]
+    pub max_zoom: u64,
```

Comment:
hard coded value in the code like previously : https://github.com/DGEXSolutions/osrd/blob/dev/chartos/chartos/settings.py#L13

Reply:
yeah that's fine